src/rpc/rawtransaction.cpp

```diff
@@ -453,19 +454,25 @@ UniValue createrawtransaction(const UniValue& params, bool fHelp)
     int nextBlockHeight = chainActive.Height() + 1;
     CMutableTransaction rawTx = CreateNewContextualCMutableTransaction(
         Params().GetConsensus(), nextBlockHeight);
-    
-    if (NetworkUpgradeActive(nextBlockHeight, Params().GetConsensus(), Consensus::UPGRADE_OVERWINTER)) {
-        if (rawTx.nExpiryHeight >= TX_EXPIRY_HEIGHT_THRESHOLD){
```

Comment:
Technically this means we are no longer checking here that the output of `CreateNewContextualCMutableTransaction()` is inside the threshold, but that's probably fine (we should be checking this in a unit test anyway).